### PR TITLE
Add rpmbuild container call

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

**Comment by scoheb** [comment]
@arilivigni @robnester-rh  @johnbieren 

Stage jobs passed using CI rpmbuild container...with declarative trigger

Please review

**Comment by scoheb** [comment]
See https://jenkins-continuous-infra.apps.ci.centos.org/blue/organizations/jenkins/ci-stage-pipeline-trigger/detail/ci-stage-pipeline-trigger/124/pipeline

and

https://jenkins-continuous-infra.apps.ci.centos.org/blue/organizations/jenkins/ci-stage-pipeline-f26/detail/ci-stage-pipeline-f26/136/pipeline

**Comment by arilivigni** [comment]
woowoo

File: config/Dockerfiles/rpmbuild/rpmbuild-test.sh [hunk 1]
```diff
@@ -1,21 +1,29 @@
 #!/bin/bash
 set +e
+set -x
```
Comment (johnbieren) [inline config/Dockerfiles/rpmbuild/rpmbuild-test.sh]: Please remove this, the logs are long enough as is and with the container functional, we shouldn't really need this to see why things failed when they do.  It helped when the pathing was all messed up, but shouldn't be needed anymore

**Comment by scoheb** [comment]
test from command line

**Comment by scoheb** [comment]
Also introduced in this PR is an artifact that describes the pod we are running on. This will give us info on the pod's containers in the event of some anomaly.

@arilivigni @johnbieren @p3ck ready for review again

Note, once we merge, I have to manually tag the images in openshift to ensure that the new images have the "stable" tag

**Comment by johnbieren** [comment]
@scoheb can we merge this?

**Pull request merged**